src/components/subscriptions/LicenseStatus.jsx

```diff
@@ -47,7 +53,7 @@ export default function LicenseStatus({ user }) {
 
 LicenseStatus.propTypes = {
   user: PropTypes.shape({
-    licenseStatus: PropTypes.string.isRequired,
-    pendingSince: PropTypes.object,
+    status: PropTypes.string.isRequired,
+    lastRemindDate: PropTypes.object,
```

Comment:
I think `lastRemindDate` will still be a string (if a datetime string)

Reply:
You are correct, but I convert it to an object before passing it as a prop [here](https://github.com/edx/frontend-app-admin-portal/pull/285/files#diff-406bdaef383e54ef2c022202985dfc45R92)